src/fosslight_dependency/_package_manager.py

```
@@ -87,122 +88,203 @@ def parse_direct_dependencies(self):
+                    ret_task = False
+                    self.set_direct_dependencies(False)
+                    logger.warning('No gradlew file exists (Skip to find dependencies relationship.).')
+                    if ret_plugin:
+                        logger.warning('Also it cannot run android-dependency-scanning plugin.')
+
+                if ret_task:
+                    current_mode = change_file_mode(cmd_gradle)
+                    if ret_alldeps:
+                        cmd = f"{cmd_gradle} allDeps"
+                        try:
+                            ret = subprocess.check_output(cmd, shell=True, encoding='utf-8')
```

Ruff (0.15.9)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)